contracts/cw1-whitelist/src/contract.rs

```diff
@@ -1,10 +1,13 @@
 use schemars::JsonSchema;
 use std::fmt;
 
+#[cfg(not(feature = "library"))]
```

Comment:
Very good point here, I think I messed up in other contracts, making this a default import.

Reply:
Yes, this triggers an `unused_imports` warning if not present, when importing as library.